containers/codespaces-linux/.devcontainer/Dockerfile

```
@@ -151,7 +151,7 @@ RUN apt-get update -yq\
     && sudo -u codespace git config --global core.editor "/home/codespace/.local/bin/git-ed.sh" \
     #
     # Alias code-insiders to code for easy terminal file editing
-    && echo $'\nif [[ $(which code-insiders) && ! $(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc \
+    && /bin/bash -c "echo $'\nif [[ $(which code-insiders) && ! $(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc" \
```

I think this fixes the escaping:
```suggestion
    && /bin/bash -c "echo $'\nif [[ \$(which code-insiders) && ! \$(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc" \
```